pkg/portal/cluster/statistics.go

```diff
@@ -0,0 +1,114 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"time"
+
+	prometheusAPI "github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+)
+
+type MetricValue struct {
+	Timestamp time.Time `json:"timestamp"`
+	Value     float64   `json:"value"`
+}
+
+type Metrics struct {
+	Name  string        `json:"metricname"`
+	Value []MetricValue `json:"metricvalue"`
+}
+
+func (c *client) Statistics(ctx context.Context, httpClient *http.Client, promQuery string, duration string, endTime time.Time, prometheusURL string) ([]Metrics, error) {
```

Comment:
It would be good to ensure that all exported types and functions are documented.

Reply:
Done.